laser_avoid_area_config:
- default_scene: true
  areas:
    area_inner:
      - name: polygon1
        value: "[[-0.09, 2.32, 0.1], [-0.56, 2.37, 0.1], [-0.54, 2.89, 0.1], [0.09, 2.98, 0.1]]"
      - value: "[[0.08, 0.62, 0.1], [0.6, -0.74, 0.1], [-0.05, -0.77, 0.1], [-0.21, 0.29, 0.1]]"
        name: polygon2
    area_out:
      - value: "[[2.04, -0.18, 0.1], [2.28, -1.64, 0.1], [1.35, -1.76, 0.1], [1.32, -0.71, 0.1]]"
        name: polygon1
      - name: polygon2
        value: "[[2.16, -2.95, 0.1], [2.23, -4, 0.1], [1.28, -4.04, 0.1], [1.22, -2.93, 0.1]]"
    area_mid:
      - value: "[[1.19, 2.03, 0.1], [1.67, 0.51, 0.1], [0.74, 0.19, 0.1], [0.39, 1.17, 0.1]]"
        name: polygon1
  name: scene_1
- areas:
    area_inner:
      - name: polygon1
        value: "[[4.49, -1.6, 0.1], [4.66, -4.62, 0.1], [3.5, -4.16, 0.1], [3.72, -2.54, 0.1]]"
    area_out:
      - name: polygon1
        value: "[[1.67, -0.79, 0.1], [1.69, -3.93, 0.1], [1.14, -2.74, 0.1], [0.67, -1.55, 0.1]]"
    area_mid:
      - value: "[[3.34, -0.57, 0.1], [3.1, -3.1, 0.1], [2.44, -3.25, 0.1], [2.67, -1.42, 0.1]]"
        name: polygon1
      - name: polygon2
        value: "[[2.96, -3.74, 0.1], [3.15, -4.57, 0.1], [2.54, -4.51, 0.1], [2.28, -3.89, 0.1]]"
  default_scene: false
  name: scene_2
- name: scene_3
  default_scene: false
  areas:
    area_mid:
      - value: "[[4.46, 0.24, 0.1], [4.15, -1.08, 0.1], [2.42, -1.41, 0.1], [2.61, 1.83, 0.1]]"
        name: polygon1
    area_inner:
      - value: "[[2.04, -2.28, 0.1], [0.35, -2.85, 0.1], [0.36, -0.68, 0.1], [1.41, -0.64, 0.1]]"
        name: polygon1
    area_out:
      - name: polygon1
        value: "[[4.1, -1.03, 0.1], [4.67, -4.68, 0.1], [3.61, -4.13, 0.1], [3.24, -2.44, 0.1]]"
      - name: polygon2
        value: "[[2.42, -1.82, 0.1], [2.67, -3.75, 0.1], [1.66, -3.85, 0.1], [1.58, -2.12, 0.1]]"